hotsos/cli.py

```diff
@@ -310,7 +310,8 @@ def cli(data_root, version, defs_path, templates_path, all_logs, debug,
                 spinner_msg = ''
             else:
                 show_spinner = not debug
-                spinner_msg = 'INFO: analysing {} '.format(drm.name)
+                sys.stdout.write('INFO: analysing {}\n'.format(drm.name))
```

Comment:
Since the spinner is only shown if `show_spinner == True` you could put this
line in an `if` statement so that we don't end up with an additional line when
usingn `--debug`.

```suggestion
                if show_spinner:
                    sys.stdout.write('INFO: analysing {}\n'.format(drm.name))
```

Reply:
Hi Nick,

I was just going to make this change but just ran it to check output beforehand.

--> ./scripts/hotsos --debug ~/downloads/sosreport-juju-1cb930-51-lxd-18-00368438-2023-09-20-lowcoqa.tar.xz 
INFO: extracting sosreport /home/bhaley/downloads/sosreport-juju-1cb930-51-lxd-18-00368438-2023-09-20-lowcoqa.tar.xz to /var/tmp/tmpxprc8280/sosreport-juju-1cb930-51-lxd-18-00368438-2023-09-20-lowcoqa
INFO: analysing sosreport /var/tmp/tmpxprc8280/sosreport-juju-1cb930-51-lxd-18-00368438-2023-09-20-lowcoqa
...

I'm going to tweak this as you said, but if we wanted the other INFO on un-packing gone with --debug will require another change, probably just passing debug to DataRootManager would be enough. I don't have an opinion either way.

Reply:
I think what "irked" me (if that's the right way of saying this :smile:) was the fact that you had to modify the regular expression in the `functest.sh`. But after closer inspection I realized that the `Spinner` does not print if it doesn't detect a tty.

I like your change though. Just don't start using a 20 character wide terminal all of a sudden 😜